sql/migrate/migrate.go

```diff
@@ -214,7 +238,11 @@ type LocalDir struct {
 // NewLocalDir returns a new the Dir used by a Planner to work on the given local path.
 func NewLocalDir(path string) (*LocalDir, error) {
 	fi, err := os.Stat(path)
-	if err != nil {
+	if err == os.ErrNotExist {
+		if err := os.MkdirAll(path, 0755); err != nil {
+			return nil, err
+		}
+	} else if err != nil {
 		return nil, err
 	}
```

Comment:
I'm not sure it's better in some way

```suggestion
	if err == os.ErrNotExist {
		err = os.MkdirAll(path, 0755)
	}
	if err != nil {
		return nil, err
	}
```

Reply:
Less explicit so I vote in favor of mine 😎